yalexs/doorbell.py

```diff
@@ -149,17 +165,24 @@ def has_subscription(self):
     async def async_get_doorbell_image(
         self, aiohttp_session: ClientSession, timeout=10
     ) -> bytes:
+        _LOGGER.debug("async_get_doorbell_image %s", self.device_name)
         response = await aiohttp_session.request(
             "get",
             self._image_url,
             timeout=timeout,
-            headers={"Authorization": self._content_token},
+            headers={"Authorization": self._content_token or ""},
         )
+        if response.status == 401:
+            _LOGGER.error(
```

Comment:
```suggestion
            _LOGGER.debug(
```

We shouldn't log as error because we are raising and expect the consumer of the api to handle this